ports/nrf/Makefile

```diff
@@ -259,6 +261,11 @@ SRC_C += $(addprefix lib/tinyusb/src/,\
 	portable/nordic/nrf5x/dcd_nrf5x.c \
 	)
 
+CFLAGS += \
+	-DCFG_TUSB_MCU=OPT_MCU_NRF5X \
```

Comment:
Instead of defining this at this level I suggest (this may not work but worth a try):
- don't add the `INC += -I../../shared/tinyusb` line above (this will prevent the default `tusb_config.h` from being included)
- add `INC += -I./drivers/usb`
- reuse the existing `drivers/usb/tusb_config.h` file and define this `CFG_TUSB_MCU` option there, and then do `#include "shared/tinyusb/tusb_config.h"` to get the remaining config from the common file

Reply:
Thanks, I had thought it was confusing to have two levels of this same header - but having the port one explicitely import the shared one does make it clearer for sure. It's certainly better to have this define in the header it belongs in rather than a global cflag.

I've made this change and yes it does work!